drivers/bridge/bridge.go

```diff
@@ -60,6 +60,7 @@ type networkConfiguration struct {
 	EnableIPv6           bool
 	EnableIPMasquerade   bool
 	EnableICC            bool
+	EnableBrProxyArp     bool
```

Comment:
Given it is already part of the bridge options, does it make sense to drop the `Br` part from the name of this option ?